elastic-apm-agent/src/test/java/co/elastic/apm/agent/premain/ExcludeJvmBootstrapCheckTest.java

```diff
@@ -78,22 +66,29 @@ void testEnvVariableNames() {
         assertThat(EXCLUDE_LIST_ENV_VARIABLE).isEqualTo(EXCLUDE_LIST_SYSTEM_PROPERTY.replace('.', '_').toUpperCase());
     }
 
-    private BootstrapCheck.BootstrapCheckResult runCheckWithConfigurationProperty(@Nullable String configPropKey, @Nullable String configPropValue,
-                                                                                  @Nullable String systemProp, String cmd) {
+    private BootstrapCheck.BootstrapCheckResult runCheckWithConfigurationProperty(@Nullable String allowlistValue,
+                                                                                  @Nullable String excludeListValue,
```

Comment:
[question] why do we still have an exclude list if the option was removed ?

Reply:
The configurable option was not removed, only the default list of exclusions